test/python/gemm/frost/test_block_scale_matmul.py

```
@@ -107,6 +107,64 @@ def _build_nvfp4_graph(
+    weight = g.tensor(dim=[1, k, n], stride=[k * n, 1, k], data_type=cudnn.data_type.FP4_E2M1)
+    weight_scales = g.tensor(
+        dim=[1, sf_k, n],
+        stride=[sf_k * n, 1, sf_k],
+        data_type=cudnn.data_type.FP8_E4M3,
+        reordering_type=cudnn.tensor_reordering.F8_128x4,
+    )
+    a = g.block_scale_dequantize(input=packed, descale=scales, block_size=[1, 16])
+    b = g.block_scale_dequantize(input=weight, descale=weight_scales, block_size=[16, 1])
+    g.matmul(A=a, B=b).set_output(True).set_data_type(cudnn.data_type.BFLOAT16)
+
+    with pytest.raises(NotImplementedError, match="call cudnn.ops.nvfp4_block_scale_quantize explicitly"):
```

Ruff (0.16.3)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)